packages/apache_tomcat/data_stream/access/manifest.yml

```
@@ -36,6 +36,22 @@ streams:
         required: false
         show_user: false
         description: Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.
+      - name: harvester_limit
+        type: integer
+        title: Harvester Limit
+        description: "Limits the number of harvesters that are started in parallel."
+        required: false
+        show_user: false
+        default: 0
+      - name: close.on_state_change.inactive
+        type: text
+        title: File Handle Closure duration
+        description: The duration after which the file handle is closed if the file has not been harvested. More details [here](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#filebeat-input-filestream-close-inactive)
```

Similar thing here, we can better explain what it means for a file not to be harvested.
```suggestion
        description: The duration after which the file handle is closed if the file has not been updated. More details [here](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#filebeat-input-filestream-close-inactive).
```